plugins/org.locationtech.udig.libs/src/org/locationtech/udig/libs/internal/PluginLogAppender.java

```diff
@@ -0,0 +1,132 @@
+/*******************************************************************************
+Copyright (c) 2005 John J. Franey
```

Comment:
Do we need to raise a CQ request at Eclipse Foundation for this copied code fragments? I'm not sure about the procedure to integrate source code from other projects ..

Reply:
I also have no idea how this works.  I thought because it was EPL it was ok, but maybe not.